notebooks/ifu_optimal/ifu_optimal.ipynb

```diff
@@ -4,18 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<p>Hi Patrick running this notebook locally and noticed that the <code>from astropy import units as u </code>  is missing from the imports.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>